go.mod

```diff
@@ -5,6 +5,7 @@ go 1.16
 require (
 	github.com/drone/envsubst v1.0.3
 	github.com/spf13/cobra v1.1.1
+	github.com/spf13/pflag v1.0.5
```

Comment:
why is this needed?

Reply:
Reading the command flags from the user

Reply:
I'm not sure it's actually needed, let's check